pkg/cmd/operator/webhooks_test.go

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2021 ScyllaDB
+
+package operator
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net"
+	"net/http"
+	"os"
+	"reflect"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+)
+
+func TestComplete(t *testing.T) {
+	now := time.Now()
+	_, cert, privateKey, _, _, err := createCertificates(now)
+	if err != nil {
+		panic(err)
+	}
+
+	ts := []struct {
```

Comment:
I'm assuming `tests`; copied from https://github.com/scylladb/scylla-operator/blob/153f6a04b15fc02d79d771e3e714c55a390aeb28/pkg/cmd/releasenotes/helpers_test.go#L10. I'm happy to change to `tt` though.

Reply:
yeah, I see `ts` as slightly unfortunate because the short variant for an item in that array would be `t` which conflicts. Same way we do the naming for `pods`->`pod`. So using `tt` and `tc` kind of avoids that implied rule while it still keeps the repeated names short.
Completely get you just took an existing one.